scheduler/clients/rust/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-nettu_scheduler_api_structs = { path = "../api_structs", version = "0.3.1" }
-nettu_scheduler_domain = { path = "../domain", version = "0.2.1" }
+nettu_scheduler_api_structs = { path = "../../crates/api_structs", version = "0.3.1" }
+nettu_scheduler_domain = { path = "../../crates/domain", version = "0.2.1" }
```

Comment:
I think we can remove all versions for internal deps from all `Cargo.toml`s

Reply:
The version numbers are needed when publishing to crates.io.

This is the error you get if there is not a version:
```
$ cd scheduler/clients/rust
$ cargo publish
    Updating crates.io index
error: all dependencies must have a version specified when publishing.
dependency `nettu_scheduler_api_structs` does not specify a version
Note: The published dependency will use the version from crates.io,
the `path` specification will be removed from the dependency declaration.
```

Reply:
ah, ok, didn't know that.